en/products/vpn/features.ftl

```diff
@@ -0,0 +1,87 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/features/
+
+vpn-features-page-title = Features - { -brand-name-mozilla-vpn }
+vpn-features-mozilla-vpn = { -brand-name-mozilla-vpn }
+
+# HTML page title
+# Line break for visual format only
+vpn-features-features-that-protect = Features that protect <br> your life online
+
+vpn-features-convenient = Convenient
+
+# Variables
+#   $servers (number) - number of VPN servers
+#   $countries (number) - number of available countries
+vpn-features-more-than = More than { $servers } servers in { $countries }+ countries
```

Comment:
```suggestion
vpn-features-more-than = { $servers -> 
    [one] More than { $servers } server in { $countries }+ countries
   *[other] More than { $servers } servers in { $countries }+ countries
}
```

Sadly, 2 plurals in a string is very complicated in our current Fluent implementation.

@peiying2 
I wonder how this is going to show up in Smartling. We need extra attention from translators.

Reply:
For this string as well as the number of devices string above, we know these will ALWAYS be numbers greater than 1. Do we still need a singular form as a matter of best practice (when number is a variable), or might these be an exception?

Reply:
> we know these will ALWAYS be numbers greater than 1

This approach is valid for "simple" languages (English, romance languages, German), but not for a lot of other languages.

For a language like [Russian](https://www.unicode.org/cldr/charts/43/supplemental/language_plural_rules.html#ru), 21 needs a different noun than 20, 22-24, or 25-29. Without knowing the number in advance, there's no way to translate it correctly (the alternative is to disconnect the number from the noun, e.g. putting it between parentheses). 

In general, when there is a variable number with a noun, [we need a plural form](https://www.unicode.org/cldr/charts/43/supplemental/language_plural_rules.html).